src/zalopay/models/disbursementQueryMerchantBalanceResponse.ts

```diff
@@ -0,0 +1,68 @@
+/*
+* The version of the ZaloPay OpenAPI document: v1.0.0
+* Contact: developer@zalopay.vn
+*
+* NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+* https://openapi-generator.tech
+* Do not edit this class manually.
+*/
+import { DisbursementQueryMerchantBalanceResponseData } from './disbursementQueryMerchantBalanceResponseData';
+
+export class DisbursementQueryMerchantBalanceResponse {
+    /**
+    * 1 - SUCCESS  2 - FAIL
+    */
+    'return_code'?: number;
```

Comment:
hmm this field should be required. Also for other fields.

I think you are misunderstood about the problem, in our request model, we require user to input `app_id`, `mac` information because the old api require to do so. However when introducing sdk, we don't really need user to input  `app_id`, `mac` as we can calculate them by ourself. Therefore @noradomi has changed them to optional.

Therefore I think we only need to set optional to fields that can be calculated by us. The rest should be keep as it is from the spec